libr/core/cmd_open.c

```diff
@@ -1413,6 +1413,8 @@ static int cmd_open(void *data, const char *input) {
 			return 0;
 		}
 		if (argv) {
+			// Unescape spaces from the path
+			r_str_replace (argv[0], "\\ ", " ", true);
```

Comment:
We should use escape and unescape apis not replace. Otherwise this replacing may be unsafe

Reply:
But at least it would be a bit better like this i think

Reply:
My initial implementation used the escaping and unescaping apis but those broke filenames with valid escapes like the cmd_i test(https://travis-ci.com/radareorg/radare2/jobs/269637812).

The only alternative is to implement a new function for path escaping and unescaping but it would be the same as only replacing the space format. Would you prefer that?

Reply:
@yossizap yes, please do it as the inline function, with making a string copy.

Reply:
Wouldn't it be wiser to just use r_str_replace_in?